tasks/orka-deploy.yaml

```diff
@@ -3,7 +3,21 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: orka-deploy
+  labels:
+    app.kubernetes.io/version: "0.1"
```

Comment:
This label is something we will want to add for versioned releases - but for master I think we should either remove the label or set to "master" (if supported) or something similar. Same goes for the other tasks